varnish/check.py

```diff
@@ -112,25 +129,25 @@ def _get_version_info(self, varnishstat_path):
 
         # Assumptions regarding varnish's version
         use_xml = True
-        version = 3
+        version = [3, 0, 0]
 
-        m1 = re.search(r"varnish-(\d+)", output, re.MULTILINE)
+        m1 = re.search(r"varnish-(\d+\.\d+\.\d+)", output, re.MULTILINE)
```

Comment:
I would still need the regex to extract the version string from the output of `varnishstat -V`, wouldn't I ?